hypothesis-python/docs/reference/strategies.rst

```diff
@@ -1,15 +1,216 @@
+.. _strategies:
+
+Strategies reference
+====================
+
+Primitives
+----------
+
+.. autofunction:: hypothesis.strategies.none
+.. autofunction:: hypothesis.strategies.nothing
+.. autofunction:: hypothesis.strategies.just
+.. autofunction:: hypothesis.strategies.booleans
+
+Numeric
+-------
+
+.. autofunction:: hypothesis.strategies.integers
```

Comment:
add a top-of-section (italic, not admonition) _See also: ref-numpy, ref-arrays, ref-pandas_.